releasetool/github.py

```diff
@@ -13,16 +13,23 @@
 # limitations under the License.
 
 import base64
+import json
 import os
 import re
-from typing import List, Sequence, Union
+import time
 
+from typing import cast, List, Sequence, Union
+
+import jwt
 import requests
+from cryptography.hazmat.backends import default_backend
 
 
 _GITHUB_ROOT: str = "https://api.github.com"
 _GITHUB_UI_ROOT: str = "https://github.com"
-_MAGIC_GITHUB_PROXY_ROOT: str = "https://magic-github-proxy.endpoints.devrel-prod.cloud.goog"
+_MAGIC_GITHUB_PROXY_ROOT: str = (
```

Comment:
Are there any comments in this code base about what the magic proxy is, what it does, and why we use it?  We should add a comment explaining why we're no longer using it.

I would not be against extirpating all references to the magic proxy, but that's probably beyond the scope of this PR.